lib/provider/camunda/CamundaPropertiesProvider.js

```diff
@@ -132,7 +132,12 @@ var getListenerLabel = function(param, translate) {
 var PROCESS_KEY_HINT = 'This maps to the process definition key.';
 var TASK_KEY_HINT = 'This maps to the task definition key.';
 
-function getIdOptions(element) {
+function getIdOptions(element, translate) {
+
+  if (is(element, 'bpmn:Participant')) {
+    return { id: 'participant-id', label: translate('Participant Id') };
```

Comment:
Should not be translated here, right? Will be translated within the component.

Reply:
Correct, that was an oversight, thanks for finding it